participants/OB1-5MB

```diff
@@ -0,0 +1,58 @@
+# 5MB by OB1
+
+## (0) Project Name and Team Slack Handles
+
+5MB by OB1
+@hoffmabc, @cpacia, @stevie zollo, @drwasho, @wolf
+
+## (1) Brief Project Description
+
+5MB is a project aimed at making package repositories available on the web through IPFS and Filecoin using Powergate.
+
+## (2) Link to Project UI
+
+https://github.com/ob1company/amzn [placeholder]
+
+## (3) What does your application/UI do?
+
+Users will be able to navigate into package repository URLs and retrieve package files directly from 5MB. 5MB will then either retrieve the content from IPFS or manage the retrieval of the content from Filecoin automatically.
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+We have several large repositories totalling around 11TB that will be going online through this project.
```

Comment:
Can you provide a bit more information on the source so that it can be added to the curated dataset list?  Alternatively - please open a separate PR to make sure it is included in the list.

Reply:
We will be using several package repositories as the data sets. Are you asking that I list out the separate data sets that will be included?

Reply:
I think it's best to submit a PR to add it to the curated datasets so that you get full credit for using public datasets.